src/sentry/workflow_engine/endpoints/validators/base/action.py

```diff
@@ -16,39 +14,19 @@
 ActionConfig = dict[str, Any]
 
 
-def validate_json_schema(value, schema):
-    try:
-        validate(value, schema)
-    except JsonValidationError as e:
-        raise ValidationError(str(e))
-
-    return value
-
-
 class BaseActionValidator(CamelSnakeModelSerializer[T], Generic[T]):
```

Comment:
just fyi, my plan is to come back in another PR to restructure this + the data condition group stuff, once we hammer down a format in this PR.